src/css/custom.css

```diff
@@ -6,34 +6,49 @@
 
 /* You can override the default Infima variables here. */
 :root {
-    --ifm-color-primary: #fe008e;
-    --ifm-color-primary-dark: #e50080;
-    --ifm-color-primary-darker: #d80079;
-    --ifm-color-primary-darkest: #b20063;
-    --ifm-color-primary-light: #ff1899;
-    --ifm-color-primary-lighter: #ff259f;
-    --ifm-color-primary-lightest: #ff4bb0;
-    --ifm-background-color: #faf9f6;
-    --ifm-code-font-size: 95%;
-    --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
-  }
-  
-  /* For readability concerns, you should choose a lighter palette in dark mode. */
-[data-theme="dark"] {
-    --ifm-color-primary: #cf00b8;
-    --ifm-color-primary-dark: #ba00a6;
-    --ifm-color-primary-darker: #b0009c;
-    --ifm-color-primary-darkest: #910081;
-    --ifm-color-primary-light: #e400ca;
-    --ifm-color-primary-lighter: #ee00d4;
-    --ifm-color-primary-lightest: #ff0ee4;
-    --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
-}
-  
+  --ifm-color-primary: #fe008e;
+  --ifm-color-primary-dark: #e50080;
+  --ifm-color-primary-darker: #d80079;
+  --ifm-color-primary-darkest: #b20063;
+  --ifm-color-primary-light: #ff1899;
+  --ifm-color-primary-lighter: #ff259f;
+  --ifm-color-primary-lightest: #ff4bb0;
+  --ifm-background-color: #faf9f6;
+  --ifm-code-font-size: 95%;
+  --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
+}
+
+/* For readability concerns, you should choose a lighter palette in dark mode. */
+[data-theme='dark'] {
+  --ifm-color-primary: #cf00b8;
+  --ifm-color-primary-dark: #ba00a6;
+  --ifm-color-primary-darker: #b0009c;
+  --ifm-color-primary-darkest: #910081;
+  --ifm-color-primary-light: #e400ca;
+  --ifm-color-primary-lighter: #ee00d4;
+  --ifm-color-primary-lightest: #ff0ee4;
+  --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
+}
+
 .custom-banner-bg {
-  background: linear-gradient(135deg, #ff008e 0%, #CF00B8 100%);
+  background: var(--ifm-background-color);
```

Comment:
I think this is the default and you can remove the entire class 🤔

Reply:
Just checked. The default is the primary color (fushia)